decor/decorators.go

```diff
@@ -266,3 +266,84 @@ func CalcPercentage(total, current, width int64) (perc int64) {
 	}
 	return int64(ceil)
 }
+
+// SpeedNoUnit returns raw I/O operation speed decorator.
+//
+//	`unitFormat` printf compatible verb for value, like "%f" or "%d"
+//
+//	`width` width reservation to apply, ignored if `DwidthSync` bit is set
+//
+//	`conf` bit set config, [DidentRight|DwidthSync|DextraSpace]
+//
+// unitFormat example:
+//
+//	"%.1f" = "1.0" or "% .1f" = "1.0"
+func SpeedNoUnit(unitFormat string, minWidth, conf int) DecoratorFunc {
+	return speed(unitFormat, 0, minWidth, conf)
+}
+
+// SpeedKibiByte returns human friendly I/O operation speed decorator,
+//
+//	`unitFormat` printf compatible verb for value, like "%f" or "%d"
+//
+//	`width` width reservation to apply, ignored if `DwidthSync` bit is set
+//
+//	`conf` bit set config, [DidentRight|DwidthSync|DextraSpace]
+//
+// unitFormat example:
+//
+//	"%.1f" = "1.0MB/s" or "% .1f" = "1.0 MB/s"
+func SpeedKibiByte(unitFormat string, minWidth, conf int) DecoratorFunc {
+	return speed(unitFormat, unitKiB, minWidth, conf)
+}
+
+// SpeedKiloByte returns human friendly I/O operation speed decorator,
+//
+//	`unitFormat` printf compatible verb for value, like "%f" or "%d"
+//
+//	`width` width reservation to apply, ignored if `DwidthSync` bit is set
+//
+//	`conf` bit set config, [DidentRight|DwidthSync|DextraSpace]
+//
+// unitFormat example:
+//
+//	"%.1f" = "1.0MB/s" or "% .1f" = "1.0 MB/s"
+func SpeedKiloByte(unitFormat string, minWidth, conf int) DecoratorFunc {
```

Comment:
Can you please rename `minWidth` to `width`, so godoc comment matches actual param name.